azkaban/__main__.py

```diff
@@ -12,6 +12,9 @@
   azkaban schedule [-jkp PROJECT] [-a ALIAS | -u URL] [-b | -m MODE]
                    [-e EMAIL ...] [-o OPTION ...] [-s SPAN] (-d DATE) (-t TIME)
                    FLOW [JOB ...]
+  azkaban sla [-a ALIAS | -u URL] (--delay=DELAY) [--rule=RULE]
+                  [-e EMAIL ...] [-k]
+                  SCHEDULE
```

Comment:
Could you document what `SCHEDULE` is in the arguments section below? It would also be great to align and move the options around for consistency:

```
azkaban sla [-k] [-a ALIAS | -u URL] [-e EMAIL ...] [--rule=RULE]
            (--delay=DELAY) SCHEDULE
```